server/index.js

```diff
@@ -1,31 +1,94 @@
-const {GraphQLServer} = require('graphql-yoga');
-const {Prisma} = require('prisma-binding');
-const {config} = require('dotenv');
-const Mutation = require('./src/resolvers/Mutation');
- const Query = require('./src/resolvers/Query');
- const User = require('./src/resolvers/User');
+const path = require('path')
+const { GraphQLServer } = require('graphql-yoga') 
+const { makePrismaSchema, prismaObjectType } = require('nexus-prisma') 
+const { prisma } = require('./src/generated/prisma-client') 
+const datamodelInfo = require('./generated/nexus-prisma') 
+const {stripe} = require('./src/stripe') 
+const { stringArg } = require('nexus/dist/definitions/args') 
 
 
+const Query = prismaObjectType({
+  name: 'Query',
+  definition(t) {t.prismaFields(['*'])}
+})
+const Mutation = prismaObjectType({
```

Comment:
Why would you need to define it on index and mutations at the same time? Wouldn't it be easier to just import it from the mutation file?